app/views/users/profile.html.erb

```diff
@@ -69,17 +71,9 @@
 
   <div>
     <i class="fas fa-user-friends"></i>
-    <% if @profile_user.followers.count>0 %>
-      <a href="/people/<%= @profile_user.username %>/followers" class="link-secondary"><%= pluralize(@profile_user.followers.count, 'follower', plural: 'followers')%></a>
-    <% else %>
-      <%= pluralize(@profile_user.followers.count, 'follower', plural: 'followers')%>
-    <% end %>
-    &#8226;    
-    <% if @profile_user.following_users.count>0 %>
-      <a href="/people/<%= @profile_user.username %>/following" class="link-secondary"><%= pluralize(@profile_user.following_users.count, 'following', plural: 'following')%></a>
-    <% else %>    
-      <%= pluralize(@profile_user.following_users.count, 'following', plural: 'following')%>
-    <% end %>
+    <a href="/people/<%= @profile_user.username %>/followers" class="link-secondary"><%= pluralize(@profile_user.followers.count, 'follower', plural: 'followers')%></a>
+    &#8226;
+    <a href="/people/<%= @profile_user.username %>/following" class="link-secondary"><%= pluralize(@profile_user.following_users.count, 'following', plural: 'following')%></a>
```

Comment:
@Forchapeatl This change shouldn't be in this PR.

Reply:
Okay , sorry